.docker/plugins/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.6
 
 RUN apk update
-RUN apk add xfsprogs e2fsprogs ca-certificates
+RUN apk add xfsprogs e2fsprogs ca-certificates nvme-cli
```

Comment:
Hi @jippi, 

Please make a specific Dockerfile for AWS and add nvm-cli there, not to the generic Dockerfile.